### PR TITLE
Fix react on older browsers

### PR DESCRIPTION
**Pull request by chantra**
- replace `for ... of` by `Array.prototype.forEach` : es6 only:
  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of
- add es5shim js library:
  http://facebook.github.io/react/docs/working-with-the-browser.html#browser-support-and-polyfills

### CODE REVIEW

**Comment by chantra** [comment]
I had to rebase on an older version of ATC, mainly because it did not seem react 0.13 was supported.

Let see if this solves the problem for others and see how we can get this on top of master.

**Comment by chantra** [comment]
this is now working on top of master!

File: atc/django-atc-demo-ui/atc_demo_ui/templates/atc_demo_ui/base.html [hunk 1]
```diff
@@ -10,6 +10,8 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
 
     <!-- React.js -->
+    <script src="//cdnjs.cloudflare.com/ajax/libs/es5-shim/2.2.0/es5-shim.min.js"></script>
+    <script src="//cdnjs.cloudflare.com/ajax/libs/es5-shim/2.2.0/es5-sham.min.js"></script>
```
Comment (zfjagann) [inline atc/django-atc-demo-ui/atc_demo_ui/templates/atc_demo_ui/base.html]: Why have two of these? They look the same.

Comment (chantra) [inline atc/django-atc-demo-ui/atc_demo_ui/templates/atc_demo_ui/base.html]: sham and shim each of them cover different functions: https://github.com/es-shims/es5-shim

Sham was not needed until react 0.13 (from my experiment)

Comment (zfjagann) [inline atc/django-atc-demo-ui/atc_demo_ui/templates/atc_demo_ui/base.html]: Yea, I just noticed the difference. LFTM!

**Pull request merged**